net.sf.VICE.yml

```
@@ -10,54 +10,144 @@ finish-args:
   - --socket=pulseaudio
   - --socket=x11
   - --metadata=X-DConf=migrate-path=/net/sf/VICE/
+add-extensions: 
+  - org.freedesktop.Platform.ffmpeg-full: 
+    directory: lib/ffmpeg
+    version: 21.08
+    add-ld-path: .
+cleanup-commands:
+  - mkdir -p /app/lib/ffmpeg  
```

```suggestion
  - --env=LD_LIBRARY_PATH=/app/lib/codecs/lib
```

Actually all you need is this.